rotkehlchen/exchanges/utils.py

```diff
@@ -32,3 +47,59 @@ def deserialize_asset_movement_address(
             value = None
 
     return value
+
+
+def create_binance_symbols_to_pair(
+    exchange_data: Dict[str, Any],
+    location: Location,
+) -> Dict[str, BinancePair]:
+    """Parses the result of 'exchangeInfo' endpoint and creates the symbols_to_pair mapping
+    """
+    result: Dict[str, BinancePair] = {}
+    for symbol in exchange_data['symbols']:
+        symbol_str = symbol['symbol']
+        if isinstance(symbol_str, FVal):
+            # the to_int here may rase but should never due to the if check above
+            symbol_str = str(symbol_str.to_int(exact=True))
+        try:
+            result[symbol_str] = BinancePair(
+                symbol=symbol_str,
+                base_asset=asset_from_binance(symbol['baseAsset']),
+                quote_asset=asset_from_binance(symbol['quoteAsset']),
+                location=location,
+            )
+        except (UnknownAsset, UnsupportedAsset) as e:
+            log.debug(f'Found pair with no processable asset. {str(e)}')
+    return result
+
+
+def query_binance_exchange_pairs(location: Location) -> Dict[str, BinancePair]:
+    db = GlobalDBHandler()
+    last_pair_check = Timestamp(
```

Comment:
this is timestamp, name it accordingly

```suggestion
    last_pair_check_ts = Timestamp(
```